scripts/build_db_from_esri.py

```diff
@@ -1786,6 +1786,10 @@ def import_hvcoordsys():
             authority = row[idx_authority]
             esri_name = row[idx_name]
 
+            if authority == 'Esri' and code in ('9897',):
```

Comment:
Sorry, missed this before merge -- but there's other instances of this I've found too. Eg 9459. I tried to get a comment from ESRI about it in this ticket (https://github.com/Esri/projection-engine-db-doc/issues/34) but their repo looks like a "dump and ignore" type job.

I'm curious on what your thoughts are on the best way to handle this. Should we continue building a mega list here of the incorrectly attributed ones?